webdriver/tests/bidi/emulation/set_locale_override/conftest.py

```diff
@@ -55,3 +55,102 @@ def another_locale(default_locale, some_locale):
     raise Exception(
         f"Unexpectedly could not find locale different from the default {default_locale} and {some_locale}"
     )
+
+
+@pytest_asyncio.fixture
+async def get_current_navigator_language(bidi_session):
+    async def get_navigator_language(context, sandbox=None):
+        result = await bidi_session.script.evaluate(
+            expression="navigator.language",
+            target=ContextTarget(context["context"], sandbox=sandbox),
+            await_promise=False,
+        )
+
+        return result["value"]
+
+    return get_navigator_language
+
+
+@pytest_asyncio.fixture
+async def get_current_navigator_languages(bidi_session):
+    async def get_navigator_languages(context, sandbox=None):
+        result = await bidi_session.script.evaluate(
+            expression="navigator.languages",
+            target=ContextTarget(context["context"], sandbox=sandbox),
+            await_promise=False,
+        )
+
+        arr = []
+        if result["type"] == "array":
```

Comment:
`navigator.languages` should always return an array I think? I would rather assert and fail early if it's not the case (or even skip the assert if we feel like this should be stable enough?)